zerver/lib/push_notifications.py

```diff
@@ -942,7 +942,7 @@ def get_message_payload_gcm(
         data.update(
             event="message",
             zulip_message_id=message.id,  # message_id is reserved for CCS
-            time=datetime_to_timestamp(message.date_sent),
+            time=message.date_sent.timestamp(),
```

Comment:
@zulip/mobile, can you confirm this is fine with the Android code that consumes it?

Reply:
Thanks for this PR! It looks like it's working toward debugging a notifications issue. 🙂

It should be fine? The app code we've written takes [a string from an FCM API](https://firebase.google.com/docs/reference/android/com/google/firebase/messaging/RemoteMessage#getData()), which we [parse as a Kotlin Long](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/to-long.html), and that's been the case for at least two years of releases. I assume that parsing would work equally well on a string like "123.45" as it does on a string like "123".

This change removes some validation (a call to `verify_UTC`); I assume that validation isn't necessary.